readonly/__init__.py

```diff
@@ -10,8 +10,14 @@
 
 from time import time
 
+import django
 from django.conf import settings
-from django.db.backends import util
+
+if django.VERSION < (1, 7):
```

Comment:
I actually prefer @artscoop style. Easier to read. Can this get merged? This library is essentially dead without it.